scripts/bb-test-bootable-image.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Check for a local cached configuration.
+if test -f /etc/buildslave; then
+    . /etc/buildslave
+fi
+
+# Custom test options will be saved in the tests directory.
+if test -f "../TEST"; then
+    . ../TEST
+fi
+
+TEST_BOOTABLE_IMAGE_SKIP=${TEST_BOOTABLE_IMAGE_SKIP:-"No"}
+if echo "$TEST_BOOTABLE_IMAGE_SKIP" | grep -Eiq "^yes$|^on$|^true$|^1$"; then
+    echo "Skipping disabled test"
+    exit 3
+fi
+
+ZPOOL=${ZPOOL:-"zpool"}
+ZFS=${ZFS:-"zfs"}
+CONSOLE_LOG="$PWD/console.log"
+OUTPUT_LOG="$PWD/output.log"
+
+case "$BB_NAME" in
+Fedora*)
+    true
+    ;;
+
+*)
+    echo "$BB_NAME unknown platform, skipping image test" >>$OUTPUT_LOG 2>&1
+    exit 18
+    ;;
+
+# Cleanup the pool and restore any modified system state.  The console log
+# is dumped twice to maximize the odds of preserving debug information.
+cleanup()
+{
+    dmesg >$CONSOLE_LOG
+    sudo -E $ZPOOL destroy -f "$TEST_BOOTABLE_IMAGE_POOL" &>/dev/null
+    sudo -E rm -f /etc/zfs/zpool.cache "$TEST_BOOTABLE_IMAGE_VDEV"
+    dmesg >$CONSOLE_LOG
```

Comment:
[user@ring2-projects scripts]$ git push
Counting objects: 4, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 428 bytes | 0 bytes/s, done.
Total 4 (delta 3), reused 0 (delta 0)
To git@github.com:Rudd-O/zfs-buildbot.git
   1b8372d..166b086  master -> master